### PR TITLE
EAP/WPE Page Updates

### PR DESCRIPTION
**Pull request by mason-at-pieces**

### CODE REVIEW

File: docs/product-highlights-and-benefits/live-context.mdx [hunk 1]
```diff
@@ -1,65 +1,87 @@
 ---
-title: Workstream Context
-description: Powered by Workstream Pattern Engine, Workstream Context can now understand recent workstream contexts, eliminating the need for manual grounding.
+title: Live Context
+description: Powered by Workstream Pattern Engine, Live Context is the world’s first Temporally Grounded Copilot that understands your workflow.
```
Comment (mack-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx:3]: "Powered by our Workstream Pattern Engine, Live Context enables the world’s first Temporally Grounded Copilot that understands your unique workflow"

File: docs/product-highlights-and-benefits/live-context.mdx [hunk 2]
```diff
@@ -1,65 +1,87 @@
...
+<Video type={'youtube'} src={'https://www.youtube.com/embed/aP8u95RTCGE?si=y4bhrmzG9KWsD1AS'}/>
+<MiniSpacer />
+
+We would like to introduce the Pieces for Developers **Workstream Pattern Engine (WPE)**, which powers the world’s first **Temporally Grounded Copilot**. Our goal is to push the limits of intelligent Copilot interactions through truly horizontal context awareness across the operating system, enabling your copilot to understand what you've been working on and keep up with the productivity demands that developers deal with every day. Together with your help, the Pieces Copilot will become the first that can understand recent context, eliminating the need for manual grounding.
```
Comment (mack-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx]: Maybe for this, we need to introduce Live Context as the feature so that when we refer to in in the subsequent paragraphs, ppl know what's up.

"We would like to introduce Live Context in Pieces for Developers, powered by our **Workstream Pattern Engine (WPE)**, which enables the world’s first **Temporally Grounded Copilot**"

File: docs/product-highlights-and-benefits/live-context.mdx [hunk 3]
```diff
@@ -1,65 +1,87 @@
...
 
 ## The Workstream Pattern Engine
+
 The workflow context you will interact with will come from the Workstream Pattern Engine, an "intelligently on" system that shadows your day-to-day work in progress journey to capture relevant workflow materials and temporally ground your Pieces Copilot Chats with relevant and recent context. Practically, this enables natural questions such as "What was I talking to Mack about this morning?" or "Explain that error I just saw," allowing your Pieces Copilot to truly extend your train of thought and let you spend less time trying to track things down or capture context and more time doing the things you love, like building amazing software.
 
 ## Getting Started with your Temporally Grounded Copilot
```
Comment (mack-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx:30]: "Getting started with Live Context" here

File: docs/product-highlights-and-benefits/live-context.mdx [hunk 4]
```diff
@@ -1,65 +1,87 @@
...
 
 ### Enabling/Disabling the WPE
 
-In order to use workstream context in your conversations with Pieces Copilot, you will need to enable the Workstream Pattern Engine. You can disable it at any time, but remember that the Copilot will not be able to use workstream context from when you had it disabled.
+In order to use live context in your conversations with Pieces Copilot, you will need to enable the Workstream Pattern Engine. You can disable it at any time, but remember that the Copilot will not be able to use live context from when you had it disabled.
```
Comment (mack-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx:34]: capitalize "Live Context" when referring to it as a proper noun

File: docs/product-highlights-and-benefits/live-context.mdx [hunk 5]
```diff
@@ -1,65 +1,87 @@
...
 - "Can you summarize the readme file from the pieces_for_x repo?"
 - "What did Sam have to say about the All Hands meeting in the GChat MLChat channel?"
 - "Generate a script in python using the function I saw on W3Schools to create a variable named xyz"
 - "Take the function from example_function.dart and add it as a method to the class in example_class.dart"
 - "What did Mark say about requests to the xyz api in slack?"
 
-### Use Cases and Tutorials
+## Use Cases and Tutorials
+
 1. Exception/error handling
```
Comment (mack-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx]: Lets add one here that's:

"5. Explaining confusing documentation"

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 4 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx:34]: updated all occurrences

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 1 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx:3]: updated

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 3 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx:30]: updated

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 2 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx]: @mack-at-pieces Can you take another look at this, it has been updated from rosie

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 5 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx]: @mack-at-pieces Can you take another look at this, it has been updated from rosie

File: docs/product-highlights-and-benefits/live-context.mdx [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+description: Powered by our Workstream Pattern Engine, Live Context enables the world’s first Temporally Grounded Copilot that understands your unique workflow
+---
+
+import {MiniSpacer} from "/src/components/Spacers";
+import Video from "/src/components/Video";
+
+# Applying Live Context to your Copilot
+
+:::info
+This feature is currently in Alpha and is now generally available to all users. It is only available to macOS & Windows users, with Linux support coming ASAP 🙏🏼.
```
Comment (tsavo-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx:12]: Currently in Beta**

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 5 above)
Comment (mack-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx]: Nice - I think there's still opportunity to add this one. Thoughts?

File: docs/product-highlights-and-benefits/live-context.mdx [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+:::
+
+Since the original launch of Pieces for Developers, we’ve been laser-focused on developer productivity. We started by giving devs a place to store their most valuable snippets of code, then moved on to proactively saving and contextualizing them. Next, we built one of the first on-device LLM-powered AI copilots in the Pieces Copilot. Now, we’re taking our obsession with contextual developer productivity to the next level with the launch of Live Context within our copilot, making it the world’s first temporally grounded copilot.
+
+<Video type={'youtube'} src={'https://www.youtube.com/embed/aP8u95RTCGE?si=y4bhrmzG9KWsD1AS'}/>
+<MiniSpacer />
+
+Live Context elevates the Pieces Copilot and allows it to understand your workflow and discuss how, where, and when you work. As with everything we do here at Pieces, this all happens entirely on-device and is available on macOS, Windows, and Linux. We truly believe the Pieces Copilot+ will revolutionize how you cross off your to-dos.
```
Comment (tsavo-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx:20]: Also want to link to the blog Georgia just released

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 2 above)
Comment (mack-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx]: Nice - I think we can still slide the "Live Context" part of the first sentence in here. Lmk thoughts.

File: docs/product-highlights-and-benefits/live-context.mdx [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+
+Pieces Copilot+ is now temporally grounded in exactly what you have been working on, which reduces the need for manual context input, enhances continuity in developer tasks, reduces context switching, and enables more natural, intuitive interactions with the Pieces Copilot.
+
+## The Workstream Pattern Engine
+
+The Live Context in the Pieces Copilot comes from the Workstream Pattern Engine, which shadows your workflow on an operating system level. Whether you're on macOS, Windows, or Linux, your workflow will be captured, processed, and stored on-device for access at a later date. The Workstream Pattern Engine applies several on-device and real-time algorithms to this data to proactively capture the most important information as you go about your workday, wherever you are— your browser, your IDE, or your collaboration tools. Over time, the Workstream Pattern Engine learns more and more about your work and becomes increasingly helpful. It even takes context from your previous copilot chats to build on the knowledge you are gathering.
```
Comment (tsavo-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx:28]: Probs want to link macos, linux, and windows here to the getting started pages for each of them

File: docs/product-highlights-and-benefits/live-context.mdx [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+
+The Workstream Pattern Engine must be turned on to use Live Context. We’ve made this super easy to do from wherever you’re getting started.
+
+:::
+
+You can add additional context to further tailor the conversation if you’d like.
+
+### Permissions
+
+If you’re a Mac user, you will need to update Pieces’ permissions in order to use Live Context. Windows users may disregard this step.
```
Comment (tsavo-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx:56]: Maybe make this a "**Note for macOS**... and don't mention windows at all. Don't wanna raise any questions explicitly around why windows can disregard

File: docs/features/pieces-copilot.mdx [hunk 10]
```diff
@@ -75,12 +75,13 @@ These snippets help the Copilot understand the precise syntax and functionality
 
 ![Setting Context for the Pieces Copilot](/assets/contextual_copilot.png)
 
-### Workstream Context
-The [Workstream Pattern Engine](/product-highlights-and-benefits/workstream-context)  enables the copilot to gather deep context from any application on your desktop, making it aware of your recent activities and work-in-progress journey. This dynamic understanding allows the copilot to provide hyper-personalized responses and suggestions based on your recent workflow.
+### Live Context
+[Live Context](/product-highlights-and-benefits/live-context) comes from the [Workstream Pattern Engine](/product-highlights-and-benefits/live-context#the-workstream-pattern-engine) and enables the copilot to gather real-time context from any application on your desktop, making it aware of your recent activities and work-in-progress journey. This dynamic understanding allows the copilot to provide hyper-personalized responses and suggestions based on your recent workflow.
```
Comment (tsavo-at-pieces) [inline docs/features/pieces-copilot.mdx:79]: instead of gather - lets use "capture"

File: docs/early-access-program.mdx [hunk 11]
```diff
@@ -3,109 +3,32 @@ title: Pieces Early Access Program
...
-As always, we've built Pieces from the ground up to put you in control. With that, **the Workstream Pattern Engine may be paused and resumed at any time**. Additionally, users may also clear locally persisted data and opt out of the Workstream Pattern Engine Early Access Program with a simple request to the team. Details on how to pause or opt-out may be found below.
+## How to Join
+If you are not currently a part of the Pieces for Developers Early Access Program, you can sign up within the Pieces Desktop App. Ensure you are signed in, then either:
+- Navigate to your `Account & Cloud Integrations` settings, or
+- Press `⌘/CTRL + Shift + P` to open the Pieces Power Menu and type `Early Access Program`.
 
-## Goals of the Early Access Program
-Our main priority for this program is to test the workstream pattern engine in as many development environments and tooling configurations as possible. From you, we want to hear pain points, any confusion you have regarding how the feature works, if you feel it adds value to your daily workflow, and **how we can make it more valuable**.
+Once you have joined, please return to this page to review how to get started with the latest features.
```
Comment (tsavo-at-pieces) [inline docs/early-access-program.mdx:24]: Screenshot of account profile with EAP badge i.e. "Once you're in you'll see this Verified EAP User Badge..."

File: docs/early-access-program.mdx [hunk 12]
```diff
@@ -3,109 +3,32 @@ title: Pieces Early Access Program
...
-
-That said, you can leverage Local Large Language Models, but this may reduce the fidelity of output and requires a fairly new machine (2021 and newer) and ideally a dedicated GPU for this. You can read [this blog](https://code.pieces.app/blog/how-to-run-an-llm-locally-with-pieces) for more information about running local models on your machine.
+## Goals of the Early Access Program
+Our main priority for this program is to test our latest features in as many development environments and tooling configurations as possible. From you, we want to hear pain points, any confusion you have regarding how features work, if you feel they add value to your daily workflow, and **how we can make them more valuable**.
 
-As always, we've built Pieces from the ground up to put you in control. With that, **the Workstream Pattern Engine may be paused and resumed at any time**. Additionally, users may also clear locally persisted data and opt out of the Workstream Pattern Engine Early Access Program with a simple request to the team. Details on how to pause or opt-out may be found below.
+## How to Join
```
Comment (tsavo-at-pieces) [inline docs/early-access-program.mdx:19]: TODO: Video on how to join EAP from within app

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 8 above)
Comment (tsavo-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx:28]: "locally captured, processed, and stored entirely on-device, where it is then available to be leveraged as Live Context within your current and future Copilot Chats"

File: docs/product-highlights-and-benefits/live-context.mdx [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+
+<Video type={'youtube'} src={'https://www.youtube.com/embed/aP8u95RTCGE?si=y4bhrmzG9KWsD1AS'}/>
+<MiniSpacer />
+
+Live Context elevates the Pieces Copilot and allows it to understand your workflow and discuss how, where, and when you work. As with everything we do here at Pieces, this all happens entirely on-device and is available on macOS, Windows, and Linux. We truly believe the Pieces Copilot+ will revolutionize how you cross off your to-dos.
+
+By using context gathered throughout your workflow, Pieces Copilot+ can provide hyper-aware assistance to guide you right back to where you left off. Ask it, “What was I working on an hour ago?” and let it help you get back into flow. Ask it, “How can I resolve the issue I got with Cocoa Pods in the terminal in IntelliJ?” or “What did Mack say I should test in the latest release?” and let Pieces Copilot surface the information that you know you have, but you can’t remember where.
```
Comment (tsavo-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx:22]: "Captured"

Gathered has a negative connotation i.e. "Government Information Gathering"

**Review by tsavo-at-pieces — COMMENTED** [review]
Couple of comments - will open issues for needed videos

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 6 above)
Comment (mack-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx:12]: @tsavo-at-pieces we currently have this in the app as "Alpha", but can adjust if needed in a patch. Lmk thoughts.

File: docs/product-highlights-and-benefits/live-context.mdx
Comment (tsavo-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx:1]: Where we say "*Local LLMs come with some hefty hardware requirements, please make sure your system can fulfill them before attempting to load one of these models.*"

Need to change this to say something more specific 


Disclaimer: Running Large Language Models locally on-device is typically resource intensive. As of right now a majority of our Local LLMs leverage roughly 7 billion parameters and to ensure optimal performance, we recommend the following minimum specifications:
- CPU: A modern multi-core processor with at least 4 cores
- RAM: At least 8GB of system memory
- GPU (Optional): A dedicated GPU with at least 4 GB of VRAM (NVIDIA TX 20 series or newer recommended)
- System Age: A machine from 2021 or newer

Please ensure your system meets these requirements before attempting to load and run these models. Inadequate hardware may result in suboptimal performance or failure to run the models."

**Review by tsavo-at-pieces — APPROVED** [review]
Small note here

File: docs/features/pieces-copilot.mdx
Comment (tsavo-at-pieces) [inline docs/features/pieces-copilot.mdx:1]: "Where we say 'Local LLMs come with some hefty hardware requirements, please make sure your system can fulfill them before attempting to load one of these models.'

Need to change this to say something more specific

Disclaimer: Running Large Language Models locally on-device is typically resource intensive. As of right now a majority of our Local LLMs leverage roughly 7 billion parameters and to ensure optimal performance, we recommend the following minimum specifications:
- CPU: A modern multi-core processor with at least 4 cores
- RAM: At least 8GB of system memory
- GPU (Optional): A dedicated GPU with at least 4 GB of VRAM (NVIDIA RTX 20 series or newer recommended)
- System Age: A machine from 2021 or newer

Please ensure your system meets these requirements before attempting to load and run these models. Inadequate hardware may result in suboptimal performance or failure to run the models."

**Review by tsavo-at-pieces — COMMENTED** [review]
Small note here

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 6 above)
Comment (mack-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx:12]: Resolution: Adjusting in-app to be Beta

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 5 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx]: Where would you say to add it @mack-at-pieces? In the `coding` list?

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 6 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx:12]: Resolved

File: docs/early-access-program.mdx (hunk 12 above)
Comment (tsavo-at-pieces) [reply docs/early-access-program.mdx:19]: Related to #427

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 7 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx]: Updated, let me know if you like where I linked the blog

File: docs/early-access-program.mdx (hunk 11 above)
Comment (tsavo-at-pieces) [reply docs/early-access-program.mdx:24]: Related to #427

File: docs/product-highlights-and-benefits/live-context.mdx [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+
+## The Workstream Pattern Engine
+
+The Live Context in the Pieces Copilot comes from the Workstream Pattern Engine, which shadows your workflow on an operating system level. Whether you're on macOS, Windows, or Linux, your workflow will be captured, processed, and stored on-device for access at a later date. The Workstream Pattern Engine applies several on-device and real-time algorithms to this data to proactively capture the most important information as you go about your workday, wherever you are— your browser, your IDE, or your collaboration tools. Over time, the Workstream Pattern Engine learns more and more about your work and becomes increasingly helpful. It even takes context from your previous copilot chats to build on the knowledge you are gathering.
+
+## Getting Started with Live Context
+
+### Enabling/Disabling the WPE
+
+In order to use Live Context in your conversations with Pieces Copilot, you will need to enable the Workstream Pattern Engine. You can disable it at any time, but remember that the Copilot will not be able to use Live Context from when you had it disabled.
```
Comment (tsavo-at-pieces) [inline docs/product-highlights-and-benefits/live-context.mdx:36]: Related to #428

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 9 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx:58]: Updated

File: docs/features/pieces-copilot.mdx (hunk 10 above)
Comment (mason-at-pieces) [reply docs/features/pieces-copilot.mdx:79]: updated

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 8 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx]: updated

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 13 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx]: I've irradicated 'gather' terms 🫡

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 2 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx]: updated

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 7 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx]: @tsavo-at-pieces

File: docs/product-highlights-and-benefits/live-context.mdx (hunk 5 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/live-context.mdx]: added to research section
